### ISSUE TITLE
obsolete chromatin silencing / reannotations to deal with

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-ontology#17160]
~chromatin silencing has a parentage 
GO:0097549    chromatin organization involved in negative regulation of transcription

but this isn't really always true. For constitutive heterochromatin it is a little indirect.

I think it should move to  "negative regulation of gene expression" (the same parent as "gene silencing".

However, it would be good to get transcription working group input on this.~

Most of this ticket is out of date. Chromatin silencing will be obsoleted.

**Comment by pgaudet** [comment]
I looked at this before, and these two terms seem pretty synonymous to me, so I didn't propose a change in ontology structure (but we should consider a merge)

https://en.wikipedia.org/wiki/Gene_silencing

**Comment by ValWood** [comment]
chromatin and gene silencing are different.  At least "gene silencing" covers all mechanisms post-transcriptional,  co-transcriptional and  chromatin silencing  (which is neither). However, we  now know more than we did 20 years ago that most of the annotations should move to some sort of "heterochromatin formation" term. These are really phenotypes established in GO before we could be sure of the mechanism. A big clean up required but I think we need experts. .....

Colin might be able to provide input here? At least for the high level which should be related directly to transcription, and which not....

**Comment by bmeldal** [comment]
Colin is @colinlog
(Welcome to the world of GitHub tickets!)

**Comment by colinlog** [comment]
To me, both facultative and constitutive heterochromatin are forms of chromatin organization of the kind that negatively regulate gene expression.
I therefore agree with the proposal that these 'chromatin silencing' annotations should be linked to a term in the ‘negative regulation of gene expression’ branch of GO (I’ll abbreviate this to ‘NRGE’). 

In fact, to me ’Chromatin silencing’ is an ambiguous word combination because it is not immediately /intuitively obvious from this terminology that it is gene expression that is silenced rather than another chromatin feature; you could eg. 'silence' the heterochromatin to obtain euchromatin? 

The closest term I can think of is indeed heterochromatin formation. NRGE can have three broad sub-classes from a transcriptional perspective; chromatin (pre-transcriptional, DNA-protein-trans-acting-RNA complexes are the actors), co-transcriptional (RNA polymerase Pre-Initiation Complex formation at promoter and onwards is the target) and post-transcriptional (I would include ‘co-transcriptional alternative splicing regulation’ in this category if it results in absence of a protein, as well as regulation of upstream ORF ATG usage leading to absence of protein product or a dominant negative product, a process that is known to play out at the ribosome level- see PMC5016099). 
In conclusion, I support moving/merging annotations from ‘chromatin silencing’ to ‘heterochromatin formation’. But, only if that is what the results that are annotated there actually describe…

**Comment by colinlog** [comment]
Sorry did not mean to close...

**Comment by ValWood** [comment]
@colinlog this is really  useful. Hopefully we can follow up on this  once the main part of transcription  is completed.

**Comment by pgaudet** [comment]
to be discussed on a transcription call

**Comment by mah11** [comment]
On a related note, should there be a link between 'heterochromatin assembly involved in chromatin silencing' (GO:0070869) and 'establishment of chromatin silencing' (GO:0006343)?

(note: I added GO:0070869, but it's been a long time since I did, so I don't really remember if the two are unconnected on purpose or just because I didn't know the silencing area well then and still don't now)

happy to move this question to its own ticket if need be

**Comment by colinlog** [comment]
Hi Midori, Val, Birdit & Pascale,

GO:0070869 Any process that results in the assembly of chromatin into
heterochromatin and contributes to chromatin silencing. *Source:* GOC:mah
GO:0006343 The initial formation of a transcriptionally silent chromatin
structure such as heterochromatin. *Source:* GOC:mah

Many of these proteins are A. nidulans annotations. The two lists are
relatively short (120 and 21 proteins) and they do not overlap.
I therefore guess that the studies these annotations come from are
independent, or that one study insisted on making a distinction, that is
not immediately obvious to me. Ultimately, it might be a distinction
between SIR complexes, H3K9me3 and H3k27me3 modification?

Overall, just on the basis of names and definitions I support merging the
two terms, and then to envisage to make 3 or 4 children for the different
flavors of heterochromatin to be annotated (H3K9me3 = constitutive
heterochromatin to silence transposons, H3K27me3 is facultative
heterochromatin to silence developmental regulators, lamin association and
the 'rubish bin' 'dark/unmodified/as yet uncharted heterochromatin' that
can only annotated to the parent term? Some histone lysin deacetylases
would fit all bins?

The heterochromatin systems are more complex than one naively would assume
because quite some loci have both K9 and K27-based heterochromatin within
one cell population!

For animal gene products, the enzymes that do H3K9me3 and H3K27me3, + the
subunits of the complexes they reside in would be annotated to
heterochromatin. Nuclear Lamin association factors for the third child?
Thomas Jenuwein and many researchers have worked on the mammalian polycomb
gene products. For Drosophila, the list of researchers involved over the
years is rather long too.

All this will become clearer once we have lists of epigenetic enzymes and
complexes that have an ascribed epigenetic Molecular Function ->  @bmeldal

Genome-wide studies that could be used to support the* DNA centric*
annotations for Heterochromatin start with Bas van Steensel (Drosophila and
human/mouse I believe). Current lists of target loci can be obtained from
groups (EBI: Adam Frankish & Daniel Zerbino, others elsewhere) who have
performed genome segmentation on the basis of epigenetic modification
combinations reported by the ENCODE as well as by the Blueprint consortia.

Hope this helps,
Colin

**Comment by mah11** [comment]
@colinlog - Thanks for all the information! I was already at (perhaps even a bit past) the limit of my understanding when I posted my earlier comment, so I appreciate the input but will have to leave sorting out the ontology to others.

Based on GO and phenotype annotations in PomBase, I would like to double-check whether this distinction:

> H3K9me3 = constitutive heterochromatin to silence transposons, H3K27me3 is facultative
heterochromatin to silence developmental regulators

... holds for all species, because for fission yeast we don't have anything about H3K27 methylation. We do have some papers* looking at H3K9 methylation in the context of facultative heterochromatin (they often monitor H3K9me2 rather than H3K9me3, but I don't know if this is biologically important or dictated by experimental convenience). Constitutive heterochromatin is found in pericentromeric regions, subtelomeres, and the silent mating-type cassette.

@ValWood, do you know whether H3K27 methylation has been observed in pombe?

*e.g. PMID:31269446, PMID:31260531

**Comment by ValWood** [comment]
hi,
yes https://www.pombase.org/reference/PMID:30773398

**Comment by ValWood** [comment]
Oh, actually this is H3`K37, not 27. `I have not come across 27...

**Comment by mah11** [comment]
yeah, we have nothing annotated to GO:0070734 or any descendant (including the MF GO:0046976), and no phenotypes mentioning H3K27

**Comment by ValWood** [comment]
At the recent fission yeast meeting a chromatin expert (Rob Martienssen) also thought that "silencing" terms were a bit "outdated" , and that we could move to more "mechanistically" described terms

We describe  by 
phenotype "silencing"
mechanism "heterochromatin assembly"
region 
pathway e.g. methylation dependent OR by small RNA (problematic because often both are involved)

and other ways...

**Comment by colinlog** [comment]
Indeed no H3K27me in Pombe as far as I could see.

The Nature paper PMID: 28682306
<https://www.ncbi.nlm.nih.gov/pubmed/28682306> is good for H3K9me2 and me3,
which are proposed there to be facultative and constitutive respectively.
Awareness of the involvement of the RNAi machinery could (?) have been the
initial observation for a distinction between
GO:0070869 Any process that results in the assembly of chromatin into
heterochromatin and contributes to chromatin silencing. *Source:* GOC:mah
and
GO:0006343 The initial formation of a transcriptionally silent chromatin
structure such as heterochromatin. *Source:* GOC:mah

However, in the above Nature paper they argue for RNAi being involved more
in the execution than in the assembly/establishment? Furthermore, the
inheritance would depend on H3K9me3.

A GO classification based on the modified residue, and subnuclear
localisation (Nuclear Lamina, perinucleolar) makes most sense to me, still.
Colin

**Comment by colinlog** [comment]
Yes this would be a better way, and if some care is taken to use the
histone residues as the distinguishing feature for the different types of
heterochromatin, the Pombe/fungal/invertebrate heterochromatin forms can be
branches of the GO that are similarly structured to those of
heterochromatin of mammals and plants. Another crucial kind of feature that
could be leading or used in definitions are the qualifiers
'peri-centromeric' and 'sub-telomeric'  heterochromatin.  But there is
quite some variation across the tree of eukaryotes on these chromosomal
chromatin features, so that is somewhat idiosynchratic. The involvement of
H3K9, which is a residue that becomes acetylated during transcription is
the common theme. H3K9 methylation inhibits the advent of H3K9 acetylation
and therefore locks the chromatin in a state that is less conducive to
transcription. Conversely, when H3K9 or H3K27 are acetylated, they cannot
be methylated without the intervention of a histone deacetylase that
removes the acetyl group thus allowing the lysine to be methylated by a
methyl-transferase.  For organisms with H3K27 methylation the logic is the
same. In fact H3K9and H3K27 both have the sequence context AR*K*S.

**Comment by ValWood** [comment]
we can close this since we no longer have the chromatin silencing terms it is out of date

**Comment by pgaudet** [comment]
This one is still there - we need to review the annotations before obsoleting.

**Comment by ValWood** [comment]
Ah right I thought all the silencing stuff was in a later ticket. This predates the obsoletion discussion.

**Comment by pgaudet** [comment]
Remaining terms in that branch: 

chromatin silencing
--
chromatin silencing at telomere
regulation of chromatin silencing
regulation of chromatin silencing at   telomere
negative regulation of chromatin   silencing
negative regulation of chromatin   silencing at telomere
positive regulation of chromatin   silencing
positive regulation of chromatin   silencing at telomere
conversion of ds siRNA to ss siRNA   involved in chromatin silencing by small RNA
siRNA loading onto RISC involved in   chromatin silencing by small RNA
heterochromatin organization involved in   chromatin silencing
production of siRNA involved in chromatin   silencing by small RNA


Are we obsoleting all ? Can we merge? (Need to look at annotations).

**Comment by ValWood** [comment]
We decided the chromatin silencing branch was not  but we did not yet discuss: 

conversion of ds siRNA to ss siRNA involved in chromatin silencing by small RNA
siRNA loading onto RISC involved in chromatin silencing by small RNA
production of siRNA involved in chromatin silencing by small RNA

 chromatin silencing ones would be, for example
GO:0031048 | heterochromatin assembly by small RNA

 However, where compound term  represent activities. So we have things like:
production of siRNA involved in chromatin silencing by small RNA
production of siRNA involved in  **RNA interference**
(so the loading/conversion/production terms are replicated in other branches)

So we would need to look at annotations and figure what the actual activities  should be 

the 'conversion conversion of ds siRNA to ss siRNA' has only 5 annotation (to argonaute and dicer).

In summary, this is a long-winded way of saying,  yes these  term are not required, but we did not discuss them yet....

**Comment by RLovering** [comment]
Hold it, I thought we were just looking at nuclear gene silencing, some of these terms are occuring at translational level eg:
GO:0036404 conversion of ds siRNA to ss siRNA is a child of RNA processing!

We need to make sure that the role of dicer and argonaute is not lost through deletion of all 'silencing' terms.

I don't know anything about chromatin silencing to be able to comment on this, I do know that some of the miRs we have curated have their action at the DNA level and transcription rather than on mRNAs.

Can we wait a bit before deleting all the terms that describe the role of RNAs in gene silencing
Thanks
Ruth

**Comment by pgaudet** [comment]
Done in other tickets
